SuperCalculator/src/supercalculator/CalculatorUtils.java

```diff
@@ -0,0 +1,46 @@
+package supercalculator;
+
+import java.util.Scanner;
+
+public class CalculatorUtils {
+
+    /**
+     * Gets standard function to prompt the user for rounding decimal across all functions and prints result, whith error check
+     *@param result  final result that a function produces
+     */
+    public  static void getOutput(double result)
+    {
+        while(true) {
+
+            System.out.println("Would you like to round the result to a given number of Decimal" +
+                    " places? please enter the number or (n) for No value must be > 0");
```

Comment:
This is a good one but I think there is point missing at the end of No. So the sentence is ......... or (n) for No. The value must be bigger than 0.

Reply:
Probably should write the second part of the sentence like this: " places? Please enter the number or (n) for No value. Must be > 0"